11-information_governance.Rmd

```diff
@@ -28,187 +28,119 @@ If requesting access to a data source, you should provide the GitHub usernames o
+> 
+> Source: [GDPR](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX:32016R0679)
 
-* __What data is being moved?__  
-* __What fields does the data contain?__
-* __What is the security marking of the data?__  
-  The Analytical Platform is suitable for data classified as OFFICIAL and OFFICIAL-SENSITIVE. SECRET and TOP SECRET data is not allowed on the Analytical Platform.
-  Information on security markings and classifying information can be found [here](https://intranet.justice.gov.uk/guidance/knowledge-information/protecting-information/classifying-information/).
-* __Will the data be modified or changed in order to mask sensitive content or personal information?__  
-  Guidance on masking and anonymisation can be found in sections \@ref(pseudonymisation) and \@ref(anonymisation).
-* __How will the data be changed or modified in order to mask sensitive content or personal information?__
-* __Will the data be modified or changed before or after being moved to the Analytical Platform?__
+Pseudonymisation may involve replacing names or other personal identifiers with reference numbers or other artificial identifiers, while maintaining a lookup enabling individuals to be re-identified.
 
-#### Personal data
+Pseudonymised data is still considered personal data and is subject to the GDPR.
 
-* __Does the data include personal data?__  
-  Personal data is information that can be used to identify a person directly or indirectly in combination with other information. Personal data includes names, identification numbers, addresses and online identifiers. Guidance from the Information Commissioner's Office (ICO) on identifying personal data can be found [here](https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/key-definitions/what-is-personal-data/). Further information on handling personal data in MoJ can be found [here](https://intranet.justice.gov.uk/guidance/knowledge-information/protecting-information/personal-data/).
-  
-#### Data controller
+You should try to anonymise or pseudonymise personal data where possible in accordance with the principles of [data minimisation](https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/principles/data-minimisation/) and [storage limitation](https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/principles/storage-limitation/).
+
+#### Data controllers
+
+When working with personal data, you should determine whether MoJ is the data controller.
+
+A data controller is an entity registered with the Information Commissioner's Office (ICO) that exercises overall control over the purposes and means of the processing of personal data. MoJ is the controller for MoJ HQ, HMPPS, HMCTS, LAA, OPG and some other agencies and public bodies.
```

Comment:
are we able to link to a list of the agencies and public bodies for which MoJ is the data controller here?

Reply:
I don't think such a list exists. This is based on what I've managed to piece together from various sources. I think this and the list in the following paragraph cover the vast majority of cases.